### ISSUE TITLE
SendRawTransaction's interface has changed

### ISSUE THREAD

**Issue by CMajeri** [ltcsuite/ltcd#18]
Hello,

One more bug:
```
                {
                    {"hexstring", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The hex string of the raw transaction"},
                    {"maxfeerate", RPCArg::Type::AMOUNT, /* default */ FormatMoney(DEFAULT_MAX_RAW_TX_FEE_RATE.GetFeePerK()),
                        "Reject transactions whose fee rate is higher than the specified value, expressed in " + CURRENCY_UNIT +
                            "/kB.\nSet to 0 to accept any fee rate.\n"},
                },
```

^ for sendrawtransaction, the second parameter used to mean "allow high fees" by setting it to "true", it now needs to be set to an amount.

**Comment by CMajeri** [comment]
This is supposed to be handled here:
```
	// Starting from bitcoind v0.19.0, the MaxFeeRate field should be used.
	case BitcoindPost19:
		// Using a 0 MaxFeeRate is interpreted as a maximum fee rate not
		// being enforced by bitcoind.
		var maxFeeRate int32
		if !allowHighFees {
			maxFeeRate = defaultMaxFeeRate
		}
		cmd = btcjson.NewBitcoindSendRawTransactionCmd(txHex, maxFeeRate)
```

but for some reason it doesn't

**Comment by CMajeri** [comment]
Ok it's because this:
```
// parseBitcoindVersion parses the bitcoind version from its string
// representation.
func parseBitcoindVersion(version string) BackendVersion {
	// Trim the version of its prefix and suffix to determine the
	// appropriate version number.
	version = strings.TrimPrefix(
		strings.TrimSuffix(version, bitcoindVersionSuffix),
		bitcoindVersionPrefix,
	)
	switch {
	case version < bitcoind19Str:
		return BitcoindPre19
	default:
		return BitcoindPost19
	}
}
```

is using wrong values to trim the prefix.
The node sends out `/LitecoinCore:0.21.2`
But the prefix expects `bitcoindVersionPrefix = "/Satoshi:"`

**Comment by CMajeri** [comment]
I suggest replacing with something like this:
```
// parseBitcoindVersion parses the bitcoind version from its string
// representation.
func parseBitcoindVersion(version string) BackendVersion {
	// Trim the version of its prefix and suffix to determine the
	// appropriate version number.
	// find the first ":" and trim everything before that
	var i int
	for i = 0; i < len(version)-1; i++ {
		if version[i] == ':' {
			break
		}
	}
	version = version[i+1:]
	switch {
	case version < bitcoind19Str:
		return BitcoindPre19
	default:
		return BitcoindPost19
	}
}
```

**Comment by losh11** [comment]
Can you open a PR with this change please?

**Comment by CMajeri** [comment]
If it's ok with you, absolutely yes

**Comment by losh11** [comment]
fixed with #20
